src/theory/datatypes/datatypes_rewriter.cpp

```diff
@@ -120,34 +120,16 @@ RewriteResponse DatatypesRewriter::postRewrite(TNode in)
     if (ev.getKind() == APPLY_CONSTRUCTOR)
     {
       Trace("dt-sygus-util") << "Rewrite " << in << " by unfolding...\n";
```

Comment:
shouldn't you update this trace?

Reply:
It's still technically unfolding, it just is multi-step unfolding :)